variation/classify.py

```diff
@@ -0,0 +1,86 @@
+"""Module for classification."""
+from typing import List
+
+from variation.classifiers import (
+    AmplificationClassifier,
+    CdnaDeletionClassifier,
+    CdnaDelInsClassifier,
+    CdnaInsertionClassifier,
+    CdnaReferenceAgreeClassifier,
+    CdnaSubstitutionClassifier,
+    GenomicDeletionAmbiguousClassifier,
+    GenomicDeletionClassifier,
+    GenomicDelInsClassifier,
+    GenomicDuplicationAmbiguousClassifier,
+    GenomicDuplicationClassifier,
+    GenomicInsertionClassifier,
+    GenomicReferenceAgreeClassifier,
+    GenomicSubstitutionClassifier,
+    GnomadVcfClassifier,
+    HgvsClassifier,
+    ProteinDeletionClassifier,
+    ProteinDelInsClassifier,
+    ProteinInsertionClassifier,
+    ProteinReferenceAgreeClassifier,
+    ProteinStopGainClassifier,
+    ProteinSubstitutionClassifier,
+)
+from variation.classifiers.classifier import Classifier
+from variation.schemas.classification_response_schema import Classification
+from variation.schemas.token_response_schema import Token, TokenType
+
+
+class Classify:
+    """The classify class."""
+
+    hgvs_classifier = HgvsClassifier()
+    gnomad_vcf_classifier = GnomadVcfClassifier()
+    classifiers: List[Classifier] = [
+        ProteinDelInsClassifier(),
+        ProteinSubstitutionClassifier(),
+        ProteinStopGainClassifier(),
+        ProteinReferenceAgreeClassifier(),
+        CdnaSubstitutionClassifier(),
+        GenomicSubstitutionClassifier(),
+        CdnaReferenceAgreeClassifier(),
+        GenomicReferenceAgreeClassifier(),
+        ProteinDelInsClassifier(),
+        CdnaDelInsClassifier(),
+        GenomicDelInsClassifier(),
+        ProteinDeletionClassifier(),
+        CdnaDeletionClassifier(),
+        GenomicDeletionClassifier(),
+        GenomicDeletionAmbiguousClassifier(),
+        ProteinInsertionClassifier(),
+        CdnaInsertionClassifier(),
+        GenomicInsertionClassifier(),
+        GenomicDuplicationClassifier(),
+        GenomicDuplicationAmbiguousClassifier(),
+        AmplificationClassifier(),
+    ]
+
+    def perform(self, tokens: List[Token]) -> Classification:
```

Comment:
this should return Optional[Classification]

Reply:
I forgot to update previous return types in perform methods (probably other methods too)